cdap-cli/pom.xml

```diff
@@ -147,6 +151,61 @@
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-resources-plugin</artifactId>
             <version>2.6</version>
+            <executions>
+              <!-- Copy CLI scripts -->
+              <execution>
+                <id>copy-cli-scripts</id>
+                <phase>prepare-package</phase>
+                <goals>
+                  <goal>copy-resources</goal>
+                </goals>
+                <configuration combine.self="override">
+                  <outputDirectory>${stage.opt.dir}</outputDirectory>
+                  <useDefaultDelimiters>false</useDefaultDelimiters>
+                  <delimiters>
+                    <delimiter>@@</delimiter>
+                  </delimiters>
+                  <resources>
+                    <resource>
+                      <directory>${project.parent.basedir}/cdap-cli/bin</directory>
+                      <targetPath>bin</targetPath>
+                      <includes>
+                        <include>cdap-cli.sh</include>
+                        <include>cdap-cli.bat</include>
+                      </includes>
+                      <filtering>true</filtering>
+                    </resource>
+                  </resources>
+                </configuration>
+              </execution>
+              <!-- Copy CLI scripts for .deb -->
```

Comment:
It's what I tried to explain in the PR description.  Without the separate one for deb, the generated ``cdap-cli.sh`` and ``cdap-cli.bat`` in the debian package do not have the ``@@project.version@@`` tokens substituted.  I believe this is because the token substitution is now done only after the "copy rpm to deb" is done